frontend/src/components/Ugur.vue

```diff
@@ -0,0 +1,122 @@
+<script>
+import { mapState, mapActions } from 'vuex'
+
+export default {
```

Comment:
Let's call this component Director — we also need to add the `name` property.

Reply:
We know this as Ugur :/ But I renamed it. 🍻